### PR TITLE
MGMT-8549- Fix missing junit files on kube api tests

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ PULL_PARAM=$(shell if [ "${CONTAINER_COMMAND}" = "podman" ];then echo "--pull-al
 
 ROOT_DIR = $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 REPORTS = $(ROOT_DIR)/reports
-
+UNITTES_JUNIT_FILE=$(shell mktemp -u "$(REPORTS)/unittest_XXXXXXXXX.xml")
```
Comment (osherdp) [inline Makefile:11]: *UNITTEST

File: Makefile [hunk 2]
```diff
@@ -489,7 +489,7 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS) _test_setup
-	JUNIT_REPORT_DIR=$(REPORTS) python3 ${DEBUG_FLAGS} -m pytest $(or ${TEST},discovery-infra/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(REPORTS)/unittest.xml
+	JUNIT_FILE=$(mktemp "$(REPORTS)/unittest_XXXXXXXXX.xml") && JUNIT_REPORT_DIR=$(REPORTS) python3 ${DEBUG_FLAGS} -m pytest $(or ${TEST},discovery-infra/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(UNITTES_JUNIT_FILE)
```
Comment (osherdp) [inline Makefile:492]:
```suggestion
	JUNIT_FILE=$(mktemp "$(REPORTS)/unittest_XXXXXXXXX.xml") JUNIT_REPORT_DIR=$(REPORTS) python3 ${DEBUG_FLAGS} -m pytest $(or ${TEST},discovery-infra/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(UNITTES_JUNIT_FILE)
```

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by eliorerz** [comment]
image
Works

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/retitle MGMT-8549- Fix missing junit files on kube api tests

**Pull request merged**